pkg/api/api.go

```diff
@@ -385,6 +386,17 @@ func (a *API) RegisterRulerAPI(r *ruler.API) {
 	a.RegisterRoute(path.Join(a.cfg.LegacyHTTPPrefix, "/rules/{namespace}"), http.HandlerFunc(r.DeleteNamespace), true, "DELETE")
 }
 
+// RegisterOverrides registers routes associated with the Overrides API
+func (a *API) RegisterOverrides(o *overrides.API) {
+	// Register individual overrides API routes with the main API
+	a.RegisterRoute("/api/v1/user-overrides", http.HandlerFunc(o.GetOverrides), true, "GET")
+	a.RegisterRoute("/api/v1/user-overrides", http.HandlerFunc(o.SetOverrides), true, "PUT")
```

Comment:
```suggestion
	a.RegisterRoute("/api/v1/user-overrides", http.HandlerFunc(o.SetOverrides), true, "POST")
```
Cortex API doesn't use PUT, for whatever reason